libs/pika/async_mpi/tests/unit/mpi_ring_async_sender_receiver.cpp

```diff
@@ -82,11 +82,13 @@ int pika_main(pika::program_options::variables_map& vm)
 
     // if comm size < 2 this test should fail
     // it needs to run on N>2 ranks to be useful
-    PIKA_TEST_MSG(size > 1, "This test requires N>1 mpi ranks");
+    //PIKA_TEST_MSG(size > 1, "This test requires N>1 mpi ranks");
```

Comment:
Should this still be commented out?

Reply:
grrrr. I keep forgetting to undo those comments when I test locally with one rank. will fix